README.md

```diff
@@ -106,6 +106,29 @@ Field's formats :
 - Sender :
 
    String
+   
+**To send sms with existing campaign**
+
+Important : To send sms with custom field, you must using tag when you creating campaign in MyElefant interface.
+
+Example : 
+
+[![Example](https://image.noelshack.com/fichiers/2019/08/4/1550760212-doc.png)](https://image.noelshack.com/fichiers/2019/08/4/1550760212-doc.png)
+
+then, in the code :  
+
+        <?php
+
+        use myelefant\MyElefant;
+
+        $client = new MyElefant(['secretKey' => '***SECRET_KEY***'])
+
+        $client->sendSms(
+                        'campaignId',
+                         ['33612345678','This content replace [[B]]','This content replace [[C]]','This content replace [[D]]']
```

Comment:
If 33612345678 is a real campaign ID, please replace with a placeholder

Reply:
No, it's a phone number